src/clients/inst_kickoff.rb

```diff
@@ -77,6 +77,13 @@ def main
         # call it always, it handles installation mode inside
         WFM.CallFunction("inst_bootloader", WFM.Args)
         #  }
+
+        # calling postpartitioning scripts in AY installation mode
+        if Mode.autoinst
```

Comment:
so this is not called during autoupgrade. What was previous state? was it called or not?